tsfresh/feature_extraction/data.py

```diff
@@ -235,6 +237,8 @@ def __init__(self, df, column_id, column_kind, column_value, column_sort=None):
 
         self.column_value = column_value
 
+        self.column_id_dtype = df[column_id].dtype
```

Comment:
Good idea with this member.
Could you also add it to the base class and describe, that it needs to be set in the implementing classes?

And how about the remaining class (the dict class)?

Reply:
Sure thing!

Regarding the `TsDictAdapter` class I left that initially as I didn't have time

Reply:
Can have a look at `TsDictAdapter`

Reply:
Was more of a general question :-) We can also support of course!